core/controlplane/config/templates/cloud-config-controller

```diff
@@ -338,14 +350,16 @@ ssh_authorized_keys:
       content: |
         [Unit]
         Description=Pull and tag a mirror image for pause-amd64
-        After=docker.service
```

Comment:
I guess doing this ends up `pause-amd64.service` to be failed (and not restarted) forever when `pause-amd64.service` "luckily" started earlier than `docker.service`?
Relates: https://github.com/kubernetes-incubator/kube-aws/pull/448/files#r107811729